modules/hcp-get-upgrade-versions.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-get-upgrade-versions_{context}"]
+= The {mce-short} hub management cluster
+
+The {mce} requires a specific {product-title} version for the management cluster to remain in a supported state. You can install the {mce-short} from OperatorHub in the {product-title} web console.
+
+See the following support matrix for the {mce-short} versions:
+
+* link:https://access.redhat.com/articles/7086906[{mce-short} 2.7]
+* link:https://access.redhat.com/articles/7073030[{mce-short} 2.6]
+* link:https://access.redhat.com/articles/7056007[{mce-short} 2.5]
+* link:https://access.redhat.com/articles/7027079[{mce-short} 2.4]
+
+The {mce-short} supports the following {product-title} versions:
+
+* The latest unreleased version
+* The latest released version
+* Two versions prior to the latest released version
+
+You can also get the {mce-short} version as a part of {rh-rhacm-first}.
+
+[id="hcp-supported-ocp-versions_{context}"]
+== Supported {product-title} versions in a hosted cluster
+
+When deploying a hosted cluster, the {product-title} version of the management cluster does not affect the {product-title} version of a hosted cluster.
+
+The HyperShift Operator creates the `supported-versions` ConfigMap in the `hypershift` namespace. The `supported-versions` ConfigMap describes the range of supported {product-title} versions that you can deploy.
+
+See the following example of the `supported-versions` ConfigMap:
+
+[source,yaml]
+----
+apiVersion: v1
+data:
+    server-version: 2f6cfe21a0861dea3130f3bed0d3ae5553b8c28b
+    supported-versions: '{"versions":["4.17","4.16","4.15","4.14"]}'
+kind: ConfigMap
+metadata:
+    creationTimestamp: "2024-06-20T07:12:31Z"
+    labels:
+        hypershift.openshift.io/supported-versions: "true"
+    name: supported-versions
+    namespace: hypershift
+    resourceVersion: "927029"
+    uid: f6336f91-33d3-472d-b747-94abae725f70
+----
+
+[IMPORTANT]
+====
+To create a hosted cluster, you must use the {product-title} version from the support version range.
```

Comment:
Can we add this? This is about the comment @dtthuynh made about MCE supporting n+1 ~ n-2 OCPs as its managed clusters.

*Note: Although the hypershift operator supports a range of OCP versions to be deployed as hosted clusters, MCE can manage only between n+1 and n-2 OCP versions. Check the MCE support matrix to ensure the hosted clusters are within the supported OCP range to be managed by MCE.